src/JsonSchema/SchemaFactory.php

```diff
@@ -20,6 +20,7 @@
 use ApiPlatform\Core\Metadata\Resource\Factory\ResourceMetadataFactoryInterface;
 use ApiPlatform\Core\Metadata\Resource\ResourceMetadata;
 use ApiPlatform\Core\Swagger\Serializer\DocumentationNormalizer;
+use Symfony\Component\HttpFoundation\Request;
```

Comment:
We don't want this part of the code to depend of HttpFoundation. Use the name of the methods as strings directly.